tests/unit/Storage/ABS/SnowflakeExportAdapterTest.php

```diff
@@ -49,7 +49,7 @@ public function testGetCopyCommand(): void
         )->willReturn($expectedCopyResult);
 
         $source = new Storage\Snowflake\Table('schema', 'table');
-        $options = new ExportOptions();
+        $options = new ExportOptions(false, ExportOptions::MANIFEST_SKIP);
```

Comment:
není skip default?

Reply:
Chtěl sem aby to bylo explicitně vidět, ten SKIP je default enom proto aby to nebylo BC.